src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -134,7 +136,6 @@ namespace picongpu
             float_X etrans = float_X(0.0);
             float_X etrans_norm = float_X(0.0);
             PMACC_CASSERT_MSG(MODENUMBER_must_be_smaller_than_number_of_entries_in_LAGUERREMODES_vector, MODENUMBER<LAGUERREMODES_t::dim);
```

Comment:
Yes, I also added at in the other method that everything is also valid if someone refactor only one of this methods.